apps/web/components/ReviewsAccordion/ReviewsAccordion.vue

```diff
@@ -17,7 +17,7 @@
           </h2>
         </template>
 
-        <UiReviewStatistics :product="product" />
+        <UiReviewStatistics :product="product" :counts="countsProductReviews" />
```

Comment:
remove this prop and move the counts logic to the component itself. Give only the product and make everything else the responsibility of the component.